contrib/README.md

```diff
@@ -207,14 +207,26 @@ The following build instructions comply with Ubuntu 20.04 and Amazon Linux 2 env
       ```
 # How to run the JDBC regression tests?
 1. Install Maven: https://maven.apache.org/install.html
-2. cd to test/JDBC
+2. Navigate to `test/JDBC`:
+
       ```
       cd test/JDBC
       ```
-3. run cleanup.sh, init.sh and mvn test
+3. Run the `cleanup.sh`, `init.sh` and `mvn test` commands:
+
       ```
       ./cleanup.sh
       ./init.sh
       mvn test
       ```
+   If you run into errors such as `psql: error: could not connect to server: No such file or directory` when invoking `init.sh` or `cleanup.sh`, try replacing the following line:
+      ```
+      psql -d postgres -U "$USER" << EOF
+      ```
+    with
+      ```
+      sudo ~/postgres/bin/psql -U "$USER" -d postgres -a << EOF
```

Comment:
1. Why do we need sudo here?
2. It depends on the user where they'll install the binaries. I think if they get the error they should just replace psql with actual path or add the binary path to PATH variable. Isn't that a better approach?

Reply:
1. I get an error when running the script without sudo, the other interns and a few people in the babelfish-opensource channel have the same issue. I thought it would be a good idea to document that sudo is a potential fix if the script does not run as is.

2. You're right, I intended for '~/postgres/bin/psql' to be 'path/to/psql/binary'.